src/app/services/tests/ConnectedUserService.spec.ts

```diff
@@ -256,9 +262,10 @@ describe('ConnectedUserService', () => {
             expect(result.isFailure()).toBeTrue();
             expect(result.getReason()).toBe('This email address is invalid.');
         });
+
         it('should fail when trying to register with a weak password', async() => {
             // Given an weak password
-            const password: string = '1';
+            password = '1';
```

Comment:
const weakPassword

Reply:
ce nombre élevé de moment ou tu dis un truc que j'ai fait littéralement 5 sec plus tôt :D
Après mieux vaut 10 commentaires et 10 réponses inutiles, explicite is safe, safe is good !